src/Gui/Splashscreen.cpp

```diff
@@ -790,6 +791,13 @@ void AboutDialog::copyToClipboard()
 
     QClipboard* cb = QApplication::clipboard();
     cb->setText(data);
+    
+    ui->copyButton->setText(tr("copied"));
+    QTimer *timer = new QTimer(this);
+    timer -> start(2000);
+    
+    connect(timer, &QTimer::timeout, [this]() { ui->copyButton->setText(tr("Copy to clipboard")); } );

```

Comment:
* I think it would look a bit neater if the text was capitalized (`Copied` or `Copied!`).
* The current way will create two identical strings that needs to be translated in sync on crowdin (one in the ui file and one here). I would prefer if the message original message was kept in a variable that you capture in the lambda context that you then revert to.
This can be done by copying the old text with `auto oldText = button->text();` and then add it to the lambda by changing `[this]` to `[this, oldText]` and use it instead of `tr("Copy to clipboard")`.

* This timer will fire every 2s, you can read how to use a singleShot timer here: https://doc.qt.io/qt-6/qtimer.html#details